lib/web_ui/lib/src/engine/font_fallbacks.dart

```diff
@@ -257,56 +265,49 @@ class FontFallbackManager {
       }
     }
 
-    // If the list of best fonts are all CJK fonts, choose the best one based
-    // on locale. Otherwise just choose the first font.
+    NotoFont? bestFontForLanguage;
     if (bestFonts.length > 1) {
+      // If the list of best fonts are all CJK fonts, choose the best one based
+      // on user preferred language. Otherwise just choose the first font.
       if (bestFonts.every((NotoFont font) =>
```

Comment:
Do we need this to check that *all* `bestFonts` are CJK, or should this kick in when there's "more than one"?

Reply:
That's a good question, and we may want to revise some of the logic in this file. But, again, this is a non-goal for this PR 🙂